charts/fulcio/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: Fulcio
 
 type: application
 
-version: 0.1.0
+version: 0.1.3
```

Comment:
We havent published the chart yet, so it can remain on `0.1.0`

Reply:
I was using terraform to deploy and test before I realised it creates a lock file and copies the charts into the scaffold dir.  But agreed, ill bump these back to 0.1.0